### ISSUE TITLE
Add safe autofix for react/no-unknown-property (class to className)

### ISSUE THREAD

**Issue by fireairforce** [utooland/utoo-lint#1269]
## Enhancement

Add a safe autofix for known React DOM property aliases in `react/no-unknown-property`, starting with `class` → `className`.

In @utoo/lint 0.4.3 the diagnostic is correct, but no fix is provided. This is a capability request, not a false-positive or false-negative report. The versioned rule-status document does not currently advertise autofix for this rule.

## Minimal reproduction

```sh
npm init -y
npm install --save-dev @utoo/lint@0.4.3
```

`example.tsx`:

```tsx
export const view=<div class='example'/>;
```

`utlint.config.json`:

```json
{
  "rules": {
    "react/no-unknown-property": "error"
  }
}
```

```sh
./node_modules/.bin/utlint --config utlint.config.json --fix-dry-run --format=json example.tsx
```

## Observed

The rule reports `Unknown property 'class' found, use 'className' instead`, with an empty `fixes` array. The preview leaves the source unchanged.

Reference comparison: ESLint 8.57.1 with eslint-plugin-react 7.37.5 and @typescript-eslint/parser 6.21.0 reports the same diagnostic and provides this fix:

```tsx
export const view=<div className='example'/>;
```

## Requested behavior

- Offer the safe attribute-name replacement through `--fix-dry-run` and `--fix`.
- Preserve the attribute value and surrounding source text.
- Respect suppression comments and avoid creating duplicate attributes when the target property already exists.

Verified on Node.js 22.23.1, macOS arm64. The issue uses only a standalone synthetic example.
